ApplicationLibCode/ProjectDataModel/Summary/RimCalculatedSummaryCase.cpp

```diff
@@ -20,9 +20,6 @@
 
 #include "RiaSummaryDefines.h"
```

Comment:
The class/file `RimCalculatedSummaryCase` class can be removed

Reply:
Fixed.